packages/vega-transforms/src/Aggregate.js

```diff
@@ -11,6 +11,7 @@ import {accessorFields, accessorName, array, error, inherits} from 'vega-util';
  * @param {Array<function(object): *>} [params.groupby] - An array of accessors to groupby.
  * @param {Array<function(object): *>} [params.fields] - An array of accessors to aggregate.
  * @param {Array<string>} [params.ops] - An array of strings indicating aggregation operations.
+ * @param {Array<object>} [params.aggregate_params=[null]] - An optional array of parameters for aggregation operations.
```

Comment:
Shouldn't this be `[params.aggregate_params]` since we cannot assume a single op?

Reply:
The code will pad aggregate_params with nulls as necessary to match the number of ops.